frontend/src/routes/FacilityBooking/ViewConflicts/ConflictBookingModal.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react'
+import { ConfirmationModal } from '../../../components/ConfirmationModal'
+import { handleCreateBooking } from '../../../store/facilityBooking/action'
+import { Booking } from '../../../store/facilityBooking/types'
+
+export default function ConflictBookingModal(props: { booking: Booking | undefined }) {
+  const [open, setOpen] = useState(false)
+  /* TODO implement function to toggle state to true */
+  function changeStatus() {
+    setOpen(!prevOpen)
+  }
+  function onRightClick() {
+    /*TODO to update createbooking function in store instead*/
+    changeStatus()
+    handleCreateBooking(props.booking?.bookingID ? true : false)
+  }
+
+  return (
+    <ConfirmationModal
+      isModalOpen={open}
+      rightButtonType="primary"
```

Comment:
button types should not be set here, confirmation modal should set them already since they are always going to be the same (right - primary, left - secondary)

Reply:
Resolved in commit 509ab25176f49a4a836ede036eb3559e1af54833.